tutorials/preprocessing/plot_80_boxy_processing.py

```diff
@@ -20,13 +20,23 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from itertools import compress
+import copy
 
 import mne
 
 
 boxy_data_folder = mne.datasets.boxy_example.data_path()
 boxy_raw_dir = os.path.join(boxy_data_folder, 'Participant-1')
-raw_intensity = mne.io.read_raw_boxy(boxy_raw_dir, verbose=True).load_data()
+raw_intensity = mne.io.read_raw_boxy(boxy_raw_dir, 'Ph', verbose=True).load_data()
+
+###separate data based on montages###
+mtg_a_indices = [i_index for i_index,i_label in enumerate(raw_intensity.info['ch_names']) if 'a01' in i_label]
+mtg_b_indices = [i_index for i_index,i_label in enumerate(raw_intensity.info['ch_names']) if 'b01' in i_label]
+mtg_a_data = copy.deepcopy(raw_intensity)
```

Comment:
raw_intensity.copy() should work (raw objects have built in copy methods)

Reply:
yeah that seems to work. I'll change that

Reply:
resolve this when you push with that change